### PR TITLE
passpie 1.3.1

### PR DESCRIPTION
**Pull request by marcwebbie**

### CODE REVIEW

File: Library/Formula/passpie.rb [hunk 1]
```diff
@@ -57,19 +57,50 @@ def install
...
+
+    # completion
+    system bin/"passpie", "-D", "passpiedb", "complete", "bash"
+    system bin/"passpie", "-D", "passpiedb", "complete", "fish"
+    system bin/"passpie", "-D", "passpiedb", "complete", "zsh"
+
+    # config
+    system bin/"passpie", "config", "global"
+    system bin/"passpie", "config", "local"
+    system bin/"passpie", "config", "current"
```
Comment (mikemcquaid) [inline Library/Formula/passpie.rb]: Could we just reduce this test to a single `passpie` invocation? Thanks!

**Comment by DomT4** [comment]
@BrewTestBot test this please

File: Library/Formula/passpie.rb (hunk 1 above)
Comment (mikemcquaid) [inline Library/Formula/passpie.rb]: Thanks!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference we prefer a single, squashed commit.

**Pull request closed without merging**